src/components/FilterBar/FilterBar.js

```diff
@@ -0,0 +1,118 @@
+
+                    <label htmlFor="one">
+                        <input type="radio" id="one" name="rating" value="1"
+                            checked={rating === "1"}
+                            onChange={(e) =>
+                                filterDispatch({ type: "FILTER_BY_RATING", payload: e.target.value })
+                            } />
+                        1 star and above
+                    </label>
+                    <label htmlFor="two">
+                        <input type="radio" id="two" name="rating" value="2"
+                            checked={rating === "2"}
+                            onChange={(e) =>
+                                filterDispatch({ type: "FILTER_BY_RATING", payload: e.target.value })
+                            } />
+                        2 star and above
+                    </label>
+                    <label htmlFor="three">
+                        <input type="radio" id="three" name="rating" value="3"
+                            checked={rating === "3"}
+                            onChange={(e) =>
+                                filterDispatch({ type: "FILTER_BY_RATING", payload: e.target.value })
+                            } />
+                        3 star and above
+                    </label>
+                    <label htmlFor="four">
+                        <input type="radio" id="four" name="rating" value="4"
+                            checked={rating === "4"}
+                            onChange={(e) =>
+                                filterDispatch({ type: "FILTER_BY_RATING", payload: e.target.value })
+                            } />
+                        4 star and above
+                    </label>
+                </div>
+                <div>
+                    <h4 className="sub-filter-header">Sort By</h4>
+                    <label htmlFor="lowToHigh">
+                        <input type="radio" id="lowToHigh" name="sortby" value="lowToHigh"
+                            onChange={() =>
+                                filterDispatch({
+                                    type: "SORT_BY_PRICE",
+                                    payload: "LOW_TO_HIGH"
+                                })
+                            }
+                            checked={sortBy === "LOW_TO_HIGH" ? true : false}
+                        />
+                        Low to High
+                    </label>
+                    <label htmlFor="highToLow">
+                        <input type="radio" id="highToLow" name="sortby" value="highToLow"
+                            onChange={() =>
+                                filterDispatch({
+                                    type: "SORT_BY_PRICE",
+                                    payload: "HIGH_TO_LOW"
+                                })
+                            }
+                            checked={sortBy === "HIGH_TO_LOW" ? true : false}
```

Comment:
You don't need to use the ternary operator here.
sortBy === "HIGH_TO_LOW"  will return either true or false by default.